xgboost_ames_housing/demo/pipelines-notebook.ipynb

```diff
@@ -0,0 +1,422 @@
+{
```

Comment:
Is this notebook contained in `ames-xgboost-build-train-deploy.ipynb` ? Do you need to check it in?